services/training/oo-design.html

```diff
@@ -0,0 +1,86 @@
+        <h4 class="margin-bottom-20">Created by Franziska Sauerwein</h4>
+        <p class="margin-bottom-40 training-description"><strong>For experienced developers that want to practise creating object oriented designs.</strong> Design objects and their relationships using the principles of OO.
+          Collaborate with a team and learn how to communicate and negotiate your ideas.</p>
+        <a href="/services#training" class="training-link-btn btn-u btn-brd btn-u-orange btn-u-lg margin-right-20"
+           type="button">More courses</a>
+        <a href="#contact" class="training-link-btn btn-u btn-brd btn-u-orange btn-u-lg" type="button">Contact</a>
+    </div>
+</div>
+<!-- End course summary -->
+
+<!-- Course description -->
+<div class="row">
+    <div class="col-md-8 col-md-offset-2">
+        <div class="row margin-bottom-20">
+            <h3>Course Duration</h3>
+	    <p>2 days</p>
+        </div>
+
+	<div class="row margin-bottom-20">
+            <h3>Why attend?</h3>
+            <ul>
+                <li>Practise the use of the most important OO design tools</li>
+                <li>Use Test Driven Development, Class-Relationship-Collaborator (CRC) cards and sequence diagrams to design</li>
+                <li>Practise separating responsibilities into objects</li>
+                <li>Assign relationships between objects to adhere to good OO design</li>
+                <li>Fill your toolbelt with different approaches to software design to be able to pick the right one according to the context</li>
+                <li>Understand and communicate benefits and drawbacks of different desing decisions</li>
+                <li>Keep the design flexible and iterate, verifying your assumptions with code</li>
+            </ul>
+        </div>
+
+        <div class="row margin-bottom-20">
+            <h3>Content</h3>
+            <ul>
+                <li>Recap of design principles</li>
+                <li>Use CRC cards as a design tool</li>
+                <li>Applying SOLID principles to object oriented design</li>
+                <li>Using sequence and class relationship diagrams to design</li>
+                <li>Using tests to drive the design</li>
+                <li>Structure problems for iterative solutions</li>
+                <li>Verify your design assumptions with code</li>
+            </ul>
+        </div>
+
+        <div class="row margin-bottom-20">
+            <h3>Audience</h3>
+            <p>Software developers that:</p>
+            <ul>
+                <li>Have experience in developing software commercially</li>
+                <li>Are familiar with at least one Object Oriented Language (such as Java and C#)</li>
+                <li>Have basic experience in Object Oriented Design</li>
+                <li>Are comfortable using TDD to build software</li>
+                <li>Understand design principles of OO such as SOLID and the 4 rules of simple design</li>
```

Comment:
Add links here to both concepts. People might know the concept but not the names. No need for them to think "ah no I don't know this, I won't take the course"